DailyCodingProblem/DailyCodingProblem/Program.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace DailyCodingProblem
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Problem1();
+        }
+
+        static void Problem1()
+        {
+            // Given a list of numbers and a number k, return whether any two numbers from the list add up to k.           
```

Comment:
Put this comment above the function as a function summary. Just put 3 forward slashes and VS will auto put the summary. (ie ///)